client/main.lua

```diff
@@ -561,6 +563,7 @@ local function init()
             for i = 1, #showroomVehicles do
                 local showroomVehicle = showroomVehicles[i]
                 local veh = createShowroomVehicle(showroomVehicle.defaultVehicle, showroomVehicle.coords)
+                table.insert(shopVehs, veh)
```

Comment:
```suggestion
                shopVehs[#shopVehs+1] = veh
```
https://manason.github.io/effective-fivem-lua/tables/#avoid-tableinsert